rust/config/mainnet_config.json

```diff
@@ -1866,6 +1866,91 @@
       },
       "validatorAnnounce": "0x5332D1AC0A626D265298c14ff681c0A8D28dB86d"
     },
+    "solana": {
+      "blockExplorers": [
+        {
+          "apiUrl": "https://api.mainnet-beta.solana.com",
+          "family": "other",
+          "name": "Solscan",
+          "url": "https://solscan.io"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "estimateBlockTime": 0.4,
+        "reorgPeriod": 0
+      },
+      "chainId": 1399811149,
+      "deployer": {
+        "name": "Abacus Works",
+        "url": "https://www.hyperlane.xyz"
+      },
+      "displayName": "Solana",
+      "displayNameShort": "Sol",
+      "domainId": 1399811149,
+      "index": {
+        "from": 259915395,
+        "mode": "sequence"
+      },
+      "interchainGasPaymaster": "9SQVtTNsbipdMzumhzi6X8GwojiSMwBfqAhS7FgyTcqy",
+      "mailbox": "75HBBLae3ddeneJVrZeyrDfv6vb7SMC3aCpBucSXS5aR",
+      "merkleTreeHook": "75HBBLae3ddeneJVrZeyrDfv6vb7SMC3aCpBucSXS5aR",
+      "name": "solana",
+      "nativeToken": {
+        "decimals": 9,
+        "name": "Sol",
+        "symbol": "SOL"
+      },
+      "protocol": "sealevel",
+      "rpcUrls": [
+        {
+          "http": "https://api.mainnet-beta.solana.com"
+        }
+      ],
+      "validatorAnnounce": ""
+    },
+    "eclipse": {
+      "blockExplorers": [
+        {
+          "apiUrl": "https://mainnetbeta-rpc.eclipse.xyz",
+          "family": "other",
+          "name": "Eclipse Explorer",
+          "url": "https://explorer.eclipse.xyz/"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "estimateBlockTime": 0.4,
+        "reorgPeriod": 0
+      },
+      "chainId": 9571022593,
+      "deployer": {
+        "name": "Abacus Works",
+        "url": "https://www.hyperlane.xyz"
+      },
+      "displayName": "Eclipse",
+      "domainId": 9571022593,
+      "index": {
+        "from": 727595,
```

Comment:
this is 1 on mainnet2, I kinda would've thought it'd be 0, but probably worth setting that https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/v2/rust/config/mainnet_config.json#L161